the riscv codepath does an include inside a namespace{} which breaks the symbols
for non-namespace use
--- a/deps/simdutf/simdutf.h
+++ b/deps/simdutf/simdutf.h
@@ -8,6 +8,8 @@
 #ifndef SIMDUTF_COMPILER_CHECK_H
 #define SIMDUTF_COMPILER_CHECK_H
 
+#include <unistd.h>
+
 #ifndef __cplusplus
 #error simdutf requires a C++ compiler
 #endif
@@ -717,7 +719,6 @@
 #elif SIMDUTF_IS_RISCV64
 
 #if defined(__linux__)
-#include <unistd.h>
 // We define these our selfs, for backwards compatibility
 struct simdutf_riscv_hwprobe { int64_t key; uint64_t value; };
 #define simdutf_riscv_hwprobe(...) syscall(258, __VA_ARGS__)
